app/javascript/src/components/Rondel.vue

```diff
@@ -148,7 +148,19 @@ export default {
       }).filter(Boolean);
     },
     nextTaxChartPosition() {
-      return nextTaxChartPosition(this.game);
+      const taxRevenue = this.game.taxRevenueOf(this.game.currentNation);
+      return this.game.getTaxChartPosition(taxRevenue);
+    },
+    playerBonus() {
+      const taxRevenue = this.game.taxRevenueOf(this.game.currentNation);
+      return this.game.playerBonusAfterUnitMaintenanceCosts(this.game.currentNation, taxRevenue);
+    },
+    nationProfit() {
+      const taxRevenue = this.game.taxRevenueOf(this.game.currentNation);
+      return this.game.nationTaxationProfit(this.game.currentNation, taxRevenue);
+    },
+    nextTaxationPowerPoints() {
+      return nextTaxationPowerPoints(this.game, this.game.currentNation);
```

Comment:
Hold on I think there's a merge conflict here that resulted in duplicated defs

Reply:
Fixed and tested in:
[7e74dee](https://github.com/Thrillberg/imperial/pull/690/commits/7e74dee9889eaa9036539725d3b20670d3b70685)

Reply:
@GitGubM8 Thanks for catching that! 👍